src/calculateRentalCost.js

```diff
@@ -28,17 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  const shortTerm = 3;
-  const longTerm = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
   const rentalPricePerDay = 40;
   const threeDaysBonus = 20;
   const sevenDaysBonus = 50;
 
-  if (days < shortTerm) {
+  if (days < SHORT_TERM) {
     return days * rentalPricePerDay;
   }
 
-  if (days < longTerm) {
+  if (days < LONG_TERM) {
     return (days * rentalPricePerDay) - threeDaysBonus;
   }
```

Comment:
You could simplify both of these IFs with a simpler return using the ternary operator (?), creating a tighter and smoother code.
Could you come up with this improvement?

Reply:
in my opinion the first option is more understandable